src/content/docs/guides/builder-escrow-proposal.mdx

```diff
@@ -0,0 +1,146 @@
+   - Internal arbitration by Smart Invoice for amounts below $1,000.
+   - Kleros arbitration for amounts exceeding $1,000.
+
+![Milestone Creation](../../../assets/images/milestone_creation.gif)
+
+### Step 4: Review and Submit
+
+1. Ensure the **`Safety Valve Date`** is at least 30 days after the final milestone.
+2. Scroll to the end of the page and select **`Add Transaction to Queue`** and wait for the confirmation **"`n` transactions queued"**.
+3. Once your transaction was queued, select **`Continue`**. You are being redirected to the next screen.
+4. Add a **`Proposal Title`** and **`Summary`** to add the actual proposal content.
+3. Click **`Submit Proposal`** to finalize the submission.
+
+Add a title and a proposal summary according to the Nouns Builder proposal template. Refer to the [proposal writing guide](https://paragraph.xyz/@nounishprof/howtoprop) and the [Nouns Builder proposal template](https://hackmd.io/@profwerder/builderdaotemplate) for further information.
+
+![Review and Submit Proposal](../../../assets/images/review_submit.png)
+
+### Release Milestone Flow
+
+Milestones can be released using the "Release Milestone" button. This action moves funds allocated to the milestone from escrow to the recipient's wallet. DAO members can vote on the milestone release proposal before execution.
+
+1. Navigate to the **Proposal Details** page to review the associated milestones.
+2. Select the milestone to release and click **"Release Milestone"**.
+3. You are being redirected to the proposal screen. Describe your deliverables and substantiate the release. Then submit the transaction.
+3. DAO voting or multisig approval is required before funds are transferred.
+
+![Milestone Release Flow](../../../assets/images/release_milestone.gif)
+
+---
+
+## Delegating Release Authority
+
+Delegation streamlines milestone management by allowing DAOs to assign fund release authority to specific entities. Attestation for this process occurs separately from the proposal builder, enabling predefined entities to securely confirm their roles and ensure transparent governance.
+
+Attestations can be made by the following entities in this order of priority
+
+1. **Builder DAO Governor (`0x6623d2a90429475ed9c8a4b613c4b5b4f8428cee`)**
+2. **Builder DAO Operations Multisig (`0x58eAEfBEd9EEFbC564E302D0AfAE0B113E42eAb3`)**
+3. **Smart Invoice Multisig (`0x503a5161D1c5D9d82BF35a4c80DA0C3Ad72d9244`)**
+
+### How Attestations Use EAS (Ethereum Attestation Service)
+
+This separate attestation flow ensures that all delegations are securely recorded, verifiable, and aligned with DAO governance protocols. If additional attestations or attesters are required for a DAO on Nouns.Builder, DAOs should contact Builder DAO or the SmartInvoice Team for assistance.
+
+#### Attestation Flow
+- Each attestation follows the schema ID `0x1289c5f988998891af7416d83820c40ba1c6f5ba31467f2e611172334dc53a0e`.
```

Comment:
@benedictvscriticus Need to add a couple points here to ensure this is clear to the DAO members. 

1. The attestation MUST be done on same chain as where dao exists;
2. The schema only has `daoMultisig` address which is what the delegated address will be and the recipient MUST be the treasury address of the dao.


cc: @wtfsayo